climakitae/core/data_export.py

```diff
@@ -52,12 +52,23 @@ def _estimate_file_size(data, format):
         est_file_size = data_size + buffer_size
 
     elif format == "CSV":
-        pass  # TODO: estimate CSV file size
-
+        # Rough estimate of the number of chars per CSV line
+        # Will overestimate uncompressed size by 10-20%
+        chars_per_line = 150
+
+        def multiply_tuple_elements(tuple):
+            result = 1
+            for item in tuple:
+                result *= item
+            return result
+
+        est_file_size = (
+            multiply_tuple_elements(data.shape) * chars_per_line
```

Comment:
I think you can do `np.prod(data.shape)` instead of `multiply_tuple_elements` for code brevity (if you wanted).

Reply:
Ahh let me try that. I was looking for a one liner

Reply:
Adopted in c00bb05. Thanks!